mne/io/eeglab/eeglab.py

```diff
@@ -347,8 +347,8 @@ def __init__(self, input_fname, montage, eog=(), event_id=None,
         info['chs'].append(stim_chan)
         info._update_redundant()
 
-        events = _read_eeglab_events(eeg, event_id=event_id,
-                                     event_id_func=event_id_func)
+        events = read_events_eeglab(eeg, event_id=event_id,
+                                    event_id_func=event_id_func)
```

Comment:
I don't understand ... the `event_id` needs to exist anyways? Otherwise the user can't pass it to the function that retrieves the events (read_raw_eeglab or read_events_eeglab).